static/app/components/modals/widgetViewerModal.tsx

```diff
@@ -156,16 +178,10 @@ function WidgetViewerModal(props: Props) {
   function renderWidgetViewer() {
     return (
       <React.Fragment>
-        {widget.queries.length > 1 && (
-          <TextContainer>
-            {t(
-              'This widget was built with multiple queries. Table data can only be displayed for one query at a time.'
-            )}
-          </TextContainer>
-        )}
         {widget.displayType !== DisplayType.TABLE && (
           <Container>
-            <WidgetCardChartContainer
+            <MemoizedWidgetCardChartContainer
+              {...props}
```

Comment:
What props are needed by the widget container? I wonder if we should explicitly pass them down instead of spreading everything that this component gets into the chart container. Is it just router/location stuff provided by the HoC in the default export?